From 1c4a41193ae9865c9c97f8e135de0bd93068a760 Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Fri, 23 May 2025 08:51:22 +0000
Subject: [PATCH] CMake: qtwaylandscanner doesn't need Wayland::Client

Remove dependency to Wayland::Client as it's not needed and
prevents building qtwaylandscanner for Windows where Client
is not supported but scanner is needed for cross-compilations.

Upstream-Status: Submitted [https://codereview.qt-project.org/c/qt/qtbase/+/647708]

Change-Id: Ia38dda58c0800799673ae2d1e29626ce6b399f04
---
 src/tools/configure.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tools/configure.cmake b/src/tools/configure.cmake
index 46d538726b3..7ab1509a89f 100644
--- a/src/tools/configure.cmake
+++ b/src/tools/configure.cmake
@@ -31,7 +31,7 @@ qt_feature("qmake" PRIVATE
         QT_FEATURE_datestring AND QT_FEATURE_regularexpression AND QT_FEATURE_temporaryfile)
 
 qt_feature("qtwaylandscanner" PRIVATE
-    CONDITION TARGET Wayland::Scanner AND TARGET Wayland::Client
+    CONDITION TARGET Wayland::Scanner
 )
 
 qt_configure_add_summary_section(NAME "Core tools")
